ui/src/state/atoms/prompt.ts

```diff
@@ -34,3 +34,13 @@ export const templateInputState = atom({
     key: "templateInput",
     default: "",
 });
+
+export const isLoggedInState = atom({
```

Comment:
Could you please create another file here with name something like auth.ts and put this state here since it is not that much related with prompt states.

This is just for file structuring purpose.